tests/rados/test_bluestore_min_alloc_size.py

```diff
@@ -133,6 +135,60 @@ def run(ceph_cluster, **kw):
                 log.info(
                     f"OSDs successfully deployed with the new alloc size, and verified the size on OSD: {osd_id}"
```

Comment:
The pre_deployment_config_changes and post_deployment_config_verification are the same. Can you create a single method to verify the pre- and post-deployment of OSDs? Please discuss this with the team.